app/models/katello/glue/pulp/repo.rb

```diff
@@ -757,8 +757,38 @@ def capsule_download_policy
         end
       end
 
+      def distributors_change?(capsule_distributors)
```

Comment:
I'll double check what it was, but there was an issue with naming it that, not sure if there is some sort of magic method thing going on. Any objection to having them both end with ```_change?``` if I run into that issue changing it back to ```_changed?``` ?